data/virt_autotest/guest_params_xml_files/sles_15_sp4_64_xen_hvm_uefi_without_power_management_x86_64_using_virtmanagerlt3.0.0.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
+<guest>
+    <guest_os_name>sles</guest_os_name>
+    <guest_version>15-sp4</guest_version>
+    <guest_version_major>15</guest_version_major>
+    <guest_version_minor>4</guest_version_minor>
+    <guest_os_word_length>64</guest_os_word_length>
+    <guest_build></guest_build>
+    <host_hypervisor_uri></host_hypervisor_uri>
+    <host_virt_type>xen</host_virt_type>
+    <guest_virt_type>hvm</guest_virt_type>
+    <guest_machine_type></guest_machine_type>
+    <guest_arch>x86_64</guest_arch>
+    <guest_name></guest_name>
+    <guest_domain_name></guest_domain_name>
+    <guest_memory>1024,maxmemory=4096</guest_memory>
```

Comment:
why does the RAM decrease to 1G from 2G which is wildly used in other tests?

Reply:
This combination '2048,maxmemory=4096' does not work for the scenario.

Reply:
how does not guest work with 2G ram?

Reply:
2G works. The combination does not work.

Reply:
Talked offline to know that '2048,maxmemory=4096' is ok in usual guest installation, while guest will fail to install with it in UEFI guest installation.